src/shady-css/tokenizer.js

```diff
@@ -25,10 +26,14 @@ class Tokenizer {
    */
   constructor(cssText) {
     this.cssText = cssText;
-    this.offset = 0;
+    this[cursorToken] = new Token(Token.type.none, 0, 0);
```

Comment:
Given the lack of class fields (sure you don't want to write this in TypeScript?), just document the `this.` assignments with JSDoc